lib/MetaProcessor/MetaParser.h

```diff
@@ -47,6 +47,7 @@ namespace cling {
   //                 StoreStateCommand := 'storeState' "Ident"
   //                 CompareStateCommand := 'compareState' "Ident"
   //                 StatsCommand := 'stats' ['ast']
+  //                 trace := 'trace' "Ident" ["Ident"]
```

Comment:
Isn't the first Ident significant, i.e. one out of `ast` (and in the future `ir`)? Then maybe put `'ast'` there, instead of `"Ident"`? Or did I misread the parser?